jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -219,8 +219,6 @@ export class MountPlugin implements IMountPlugin {
               <Datum
                 showDatum={showDatum ? showDatum : this._showDatum}
                 setShowDatum={this.setShowDatum}
-                keepMounted={this._keepMounted}
```

Comment:
this is because things will stay mounted by default, right? as they are real files on the fs now

Reply:
That's right, since we have contents managers for both explore and test there is no longer a need to unmount/remount the mounted datum when switching back to explore.